modules/nested-elements/widgets/nested-tabs.php

```diff
@@ -0,0 +1,441 @@
+<?php
```

Comment:
On drag a widget to an empty tab - it dragged without a container

Reply:
What do you mean?